src/fields.ts

```diff
@@ -179,13 +265,19 @@ export type RTBlockNode =
 	| RTImageNode
 	| RTEmbedNode;
 
-// Inline nodes
+/**
+ * Inline Rich Text Nodes
+ */
 export type RTInlineNode = RTStrongNode | RTEmNode | RTLabelNode | RTLinkNode;
 
-// All nodes
+/**
+ * All Rich Text nodes
+ */
 export type RTAnyNode = RTBlockNode | RTInlineNode | RTSpanNode;
 
-// Title field nodes
+/**
+ * All Rich Text Title Nodes
+ */
```

Comment:
For each field type (`TitleField`, `ImageField`, etc.), can we link to the Prismic Core Concepts page if it has one?

```
@see Title field documentation: {@link https://prismic.io/docs/core-concepts/rich-text-title}
```